CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributing
+
+Although we are always happy to make improvements, we also
+welcome changes and improvements from the community!
+
+Have a fix for a problem you've been running into or an idea for a new feature
+you think would be useful? Here's what you need to do:
+
+1. [Read and understand the Code of Conduct](#code-of-conduct).
+2. Fork this repo (unless you work at ackama) and clone that repo to somewhere on your machine.
+3. [Ensure that you have a working environment](#setting-up-your-environment).
+4. Read up on the [architecture of the project](#architecture),
+   [how to run tests](#tests), and [the code style we use in this project](#code-style).
+5. Cut a new branch and write a failing test for the feature or bugfix you plan
+   on implementing.
+6. [Make sure your branch is well managed as you go along](#managing-your-branch).
+7. If this branch fixes an issue, use github's `fixes: #issue_number`
+   when writing your commit message to link the issue to your pr.
+8. Push to your fork and create a pull request.
+9. Ensure that the test suite passes CI and make any necessary changes to your branch to bring it to green.
+
+We will review pull requests fortnightly at our ackama ruby guild meeting.
```

Comment:
```suggestion
We will review pull requests fortnightly at our Ackama ruby guild meeting.
```

🐘 I feel like this is slightly more relaxed (and I guess true, since we don't strictly go through all the PRs)

```suggestion
We aim to review pull requests fortnightly at our Ackama ruby guild meeting.
```